server/controllers/v1/recipeController.js

```diff
@@ -85,3 +85,34 @@ export const voteRecipe = (req, res) => {
     }
   });
 };
+
+export const addRecipeAsFavorite = (req, res) => {
+  FavoriteRecipe.create({
+    userId: req.user.id,
+    recipeId: req.params.recipeId
+  })
+    .then(favoriteRecipe => res.status(201).json({ favoriteRecipe,
+      message: 'Recipe has been favorited' }))
+    .catch((error) => {
+      if (error.name === 'SequelizeUniqueConstraintError') {
+        res.status(400).json({ message: 'Recipe has already been favorited' });
+      } else {
+        res.status(400).json({ error });
+      }
+    });
+};
+
```

Comment:
I will like to see comments in your codebase, you are not adding comments where necessary